protocol/contracts/oracle/Cargo.toml

```diff
@@ -11,9 +11,11 @@ license.workspace = true
 crate-type = ["cdylib", "rlib"]
 
 [features]
-default = ["cosmwasm-bindings"]
+default = []
```

Comment:
why have you decided to remove the default features for some contracts while still keeping them for the others?